src/modules/missionHelper/i18n/it_IT.json

```diff
@@ -22,50 +22,42 @@
 		"fire_aviation_count": "Firefighting plane station | Firefighting plane stations",
-		"fire_stations": "Firestation | Firestations",
+		"fire_stations": "Caserma VVF | Caserme VVF",
 		"max_police_stations": "Policestation maximal | Policestations maximal",
 		"max_rescue_stations": "Rescuestation maximal | Rescuestation maximal",
-		"police_helicopter_stations": "Police Helicopter Station | Police Helicopter Stations",
-		"police_stations": "Police Station | Police Stations",
-		"rescue_stations": "Ambulance Station | Ambulance Stations",
-		"title": "Prerequisite | Prerequisites"
+		"police_helicopter_stations": "Stazione elicottero polizia | Stazioni elicottero polizia",
+		"police_stations": "Stazione di Polizia | Stazioni di Polizia",
+		"rescue_stations": "Stazione ambulanza | Stazioni ambulanze",
+		"title": "Prerequisito | Prerequisitp"
 	},
 	"prisoners": {
 		"max_possible_prisoners": "Massimo prigioniere | Prigionieri massimali",
 		"title": "Prigionieri | {n} Prigioniere | {n} Prigionieri"
 	},
 	"requirements": {
-		"water_needed": "Water demand (gal)"
+		"water_needed": "Richiesta d'acqua (gal)"
 	},
 	"vehicles": {
 		"captions": {
-			"ambulances": "Ambulances",
-			"arff": "ARFF",
-			"battalion_chief_vehicles": "Battalion Chief Vehicle",
-			"boats": "Boat",
-			"brush_truck": "Wildland fire engines",
-			"fbi": "FBI Units",
-			"fbi_bomb": "FBI Bomb Technician Vehicle",
-			"fbi_drone": "FBI Surveillance Drone",
-			"fbi_investigation": "FBI Investiogation Wagon",
-			"fbi_mcc": "FBI Mobile Command Centers Vehicle",
-			"fire_aviation": "Airborne Firefighting",
-			"firetrucks": "Firetruck | Firetrucks",
-			"hazmat_vehicles": "Hazmat",
-			"heavy_rescue_vehicles": "Heavy Rescue",
-			"helicopter": "HEMS",
-			"k9": "K-9",
-			"large_fire_boat": "Large Fireboat",
-			"large_rescue_boat": "Large Rescueboat",
-			"mobile_air_vehicles": "Mobile air",
-			"mobile_command_vehicles": "Mobile Command Vehicle",
-			"platform_trucks": "Platform Truck | Platform Trucks",
-			"police_cars": "Patrol Car",
-			"police_helicopters": "Policehelicopter",
-			"sheriff": "Sheriff",
-			"swat_armored_vehicle": "SWAT Armoured Vehicle",
-			"swat_suv": "SWAT SUV",
-			"water_tankers": "Water Tanker"
+			"ambulances": "Ambulanza Medicalizzata",
+			"arff": "Flotta antincendio aereporti",
+			"battalion_chief_vehicles": "Funzionari",
+			"fbi_bomb": "Furgone artificieri",
+			"firetrucks": "APS/ABP",
+			"hazmat_vehicles": "NBCR",
+			"heavy_rescue_vehicles": "Unità polisoccorso",
+			"helicopter": "Elisoccorso",
+			"k9": "Unità cinofila antidroga",
+			"mobile_air_vehicles": "Carro aria",
+			"mobile_command_vehicles": "UCL",
+			"platform_trucks": "Autoscala | Autoscale",
+			"police_cars": "Pattuglie",
+			"police_helicopters": "Elicottero della polizia",
+			"swat_armored_vehicle": "Furgone Antisommossa",
+			"swat_suv": "Suv UOPI",
+			"water_tankers": "Kilolitriche"
+			"fwk": "Autogrù"
```

Comment:
Just to have it as a review:
Add a `,` at the end of each of both lines 58 & 59 as @Piet2001 already mentioned ^^